tests/Export/Adapters/AttributeAdapterTest.php

```diff
@@ -0,0 +1,830 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Export;
```

Comment:
This needs to be changed in all adapter test files

```suggestion
namespace FINDOLOGIC\FinSearch\Tests\Export\Adapters;
```